### PR TITLE
Adding a GNU GPLv3 license

### PR DESCRIPTION
**Pull request by NovusEdge**
Resolves Issue: #12

### CODE REVIEW

**Comment by NovusEdge** [comment]
@niloysikdar Don't merge the PR yet, there are some changes I want you to make to the file

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,674 @@
...
+  If you develop a new program, and you want it to be of the greatest
+possible use to the public, the best way to achieve this is to make it
+free software which everyone can redistribute and change under these terms.
+
+  To do so, attach the following notices to the program.  It is safest
+to attach them to the start of each source file to most effectively
+state the exclusion of warranty; and each file should have at least
+the "copyright" line and a pointer to where the full notice is found.
+
+    <one line to give the program's name and a brief idea of what it does.>
```
Comment (niloysikdar) [inline LICENSE:634]: Please add proper information here

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,674 @@
...
+    Copyright (C) <year>  <name of author>
```
Comment (niloysikdar) [inline LICENSE:635]: Please add proper information here

**Review by niloysikdar — CHANGES_REQUESTED** [review]

**Review by NovusEdge — COMMENTED** [review]
@niloysikdar 

I pointed out some changes in the review in form of comments, please do look into it....

File: LICENSE (hunk 1 above)
Comment (NovusEdge) [reply LICENSE:634]: Kindly provide the required info to be filled in.

File: LICENSE (hunk 2 above)
Comment (NovusEdge) [reply LICENSE:635]: Kindly provide the required info to be filled in.

[the remark this answers is not in the document]
Comment (niloysikdar) [reply LICENSE:635]: - Open Source Portfolio website template
- 2021
- Niloy Sikdar

File: LICENSE [hunk 3]
```diff
@@ -0,0 +1,674 @@
...
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+Also add information on how to contact you by electronic and paper mail.
+
+  If the program does terminal interaction, make it output a short
+notice like this when it starts in an interactive mode:
+
+    <program>  Copyright (C) <year>  <name of author>
```
[the remark this answers is not in the document]
Comment (niloysikdar) [reply LICENSE:655]: - Open Source Portfolio website template
- 2021
- Niloy Sikdar

**Comment by niloysikdar** [comment]
@NovusEdge , I have provided the required data, please add them and we are good to go, cheers.

**Comment by NovusEdge** [comment]
@niloysikdar 
I have made the necessary changes, please review them and let me know if something is out of order.

**Review by niloysikdar — APPROVED** [review]

**Comment by niloysikdar** [comment]
Awesome work @NovusEdge , thanks for your contribution.

**Pull request merged**